ci/test/02_run_container.sh

```diff
@@ -52,7 +56,7 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
                   --mount "type=volume,src=${CONTAINER_NAME}_depends_sources,dst=$DEPENDS_DIR/sources" \
                   --mount "type=volume,src=${CONTAINER_NAME}_depends_SDKs_android,dst=$DEPENDS_DIR/SDKs/android" \
                   --mount "type=volume,src=${CONTAINER_NAME}_previous_releases,dst=$PREVIOUS_RELEASES_DIR" \
-                  --env-file /tmp/env \
+                  --env-file /tmp/env-$USER \
```

Comment:
I don't think you're supposed to do that. One user == one worker.

If we want to support multiple workers per user, we also have to randomise the container names, since you might be running CI for two pull requests at the same time.

Reply:
> Could be a separate pull request, because this is a bug fix?

I might split it out if this PR takes too long on its own.

Reply:
Ok, so maybe `/tmp/env-$USER-$CONTAINER_NAME`